examples/tracer-web/README.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument your JavaScript code running in the browser.
```

Comment:
`@opentelemetry/web` not `@opentelemetry/tracing`

Reply:
```suggestion
This example shows how to use [@opentelemetry/web](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-web) to instrument your JavaScript code running in the browser.
```

Reply:
Fixed!